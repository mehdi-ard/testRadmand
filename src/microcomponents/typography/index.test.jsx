import React from 'react';
import renderer from 'react-test-renderer';
import Typography from './';

it('title', () => {
  expect(
    renderer
      .create(
        <Typography variant="title" level={2}>
          Hello Button
        </Typography>
      )
      .toJSON()
  ).toMatchSnapshot();
});

it('text', () => {
  expect(
    renderer
      .create(<Typography variant="text">Hello Button</Typography>)
      .toJSON()
  ).toMatchSnapshot();
});

it('Paragraph', () => {
  expect(
    renderer
      .create(<Typography variant="paragraph">Hello Button</Typography>)
      .toJSON()
  ).toMatchSnapshot();
});

it('Paragraph copyable', () => {
  expect(
    renderer
      .create(
        <Typography variant="paragraph" copyable>
          Hello Button
        </Typography>
      )
      .toJSON()
  ).toMatchSnapshot();
});
