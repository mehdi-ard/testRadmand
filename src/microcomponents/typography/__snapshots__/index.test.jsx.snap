// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph 1`] = `
<div
  className="ant-typography"
  style={
    Object {
      "WebkitLineClamp": null,
    }
  }
>
  Hello Button
</div>
`;

exports[`Paragraph copyable 1`] = `
<div
  className="ant-typography"
  style={
    Object {
      "WebkitLineClamp": null,
    }
  }
>
  Hello Button
  <div
    aria-label="Copy"
    className="ant-typography-copy"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    role="button"
    style={
      Object {
        "background": "transparent",
        "border": 0,
        "display": "inline-block",
        "lineHeight": "inherit",
        "padding": 0,
      }
    }
    tabIndex={0}
  >
    <span
      aria-label="copy"
      className="anticon anticon-copy"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="copy"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z"
        />
      </svg>
    </span>
  </div>
</div>
`;

exports[`text 1`] = `
<span
  className="ant-typography"
  style={
    Object {
      "WebkitLineClamp": null,
    }
  }
>
  Hello Button
</span>
`;

exports[`title 1`] = `
<h2
  className="ant-typography"
  style={
    Object {
      "WebkitLineClamp": null,
    }
  }
>
  Hello Button
</h2>
`;
